samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -297,3 +300,101 @@ SnapToCenterVirtualGridList.storyName = 'Snap to center VirtualGridList';
 SnapToCenterVirtualGridList.parameters = {
 	propTables: [Config]
 };
+
+// eslint-disable-next-line enact/prop-types
+const renderItemWithRandomSrc = ({index, ...rest}) => {
+	const
+		color = Math.floor((Math.random() * (0x1000000 - 0x101010)) + 0x101010).toString(16),
+		source = {
+			'hd': `http://placehold.it/200x200/${color}/ffffff&text=Image ${index}`,
+			'fhd': `http://placehold.it/300x300/${color}/ffffff&text=Image ${index}`,
+			'uhd': `http://placehold.it/600x600/${color}/ffffff&text=Image ${index}`
+		},
+		text = `Item ${index}`;
+
+	return (
+		<ImageItem {...rest} label={text} src={source} />
+	);
+};
+
+const VirtualGridListInScroller = ({onNext, ...rest}) => {
+	const virtualGridListProps = {
+		dataSize: updateDataSize(number('dataSize', Config, defaultDataSize)),
+		direction: 'horizontal',
+		itemRenderer: renderItemWithRandomSrc,
+		itemSize: {
+			minWidth: ri.scale(number('minWidth', Config, 688)),
+			minHeight: ri.scale(number('minHeight', Config, 570))
+		},
+		spacing: ri.scale(number('spacing', Config, 0)),
+		style: {
+			height: ri.scale(number('minHeight', Config, 570)),
+			paddingBottom: ri.scaleToRem(36)
+		}
+	};
+
+	const virtualGridLists = [];
+
+	for (let i = 0; i < 4; i++) {
+		const id = `vgl_${i}`;
+
+		virtualGridLists.push(
+			<VirtualGridList
+				{...rest}
+				{...virtualGridListProps}
+				childProps={{onClick: onNext}}
```

Comment:
How about put this in `virtualGridListProps`?

Reply:
It's fixed.